go.mod

```diff
@@ -17,7 +18,7 @@ require (
 	github.com/containers/image/v5 v5.16.0
 	github.com/containers/ocicrypt v1.1.2
 	github.com/containers/psgo v1.7.1
-	github.com/containers/storage v1.36.0
+	github.com/containers/storage v1.36.1-0.20220109140301-01a26f857602
```

Comment:
Why are we bumping to a commit and not to a specific tag/version?

Reply:
It's a bug fix that I was trying to get through quickly on a weekend.  I probably should have created a new release.  @jnovy what is the absolute drop dead time for this?

Reply:
I fear that the quick-and-dirty could cause confusion in the future. I am OK to merge it as is as long as we cut a release right after and bump it here.